client/src/components/Header/Header.js

```diff
@@ -1,10 +1,11 @@
-import { Grid, makeStyles } from '@material-ui/core'
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Button from '@material-ui/core/Button';
+import { Grid, makeStyles } from '@material-ui/core';
 
 const useStyles = makeStyles(theme => ({
   root:{
-    position: 'absolute',
+    // position: 'absolute', -- Temporary Change
```

Comment:
Remember not to leave these behind when committing changes. A good practice to remember these would be to use that comment you wrote "temporary change" and do a search for all instances of that in your project and remove those prior to doing your commit.